### PR TITLE
[docs][dagster-dbt] Update schedule examples

### PR DESCRIPTION
**Pull request by OwenKephart**
## Summary & Motivation

We've updated these APIs, let's update the docs. Decided to be explicit here about how you would deal with the simple (dbt-only) case, as well as the case where you want to combine dbt and non-dbt assets, as I think both are valuable.

### CODE REVIEW

File: docs/content/integrations/dbt/reference.mdx [hunk 1]
```diff
@@ -102,28 +102,39 @@ defs = Definitions(
...
+from dagster_dbt import build_dbt_asset_selection
+
+# selects all models tagged with "daily", and all their downstream asset dependencies
+daily_selection = build_dbt_asset_selection(
+    [my_dbt_assets], dbt_select="tag:daily"
+).downstream()
+
+daily_dbt_assets_and_downstream_schedule = ScheduleDefinition(
+    job=define_asset_job("daily_assets", selection=daily_selection),
+    cron_schedule="@daily",
```
Comment (rexledesma) [inline docs/content/integrations/dbt/reference.mdx:137]: Can we add this example to the `build_dbt_asset_selection` API docs?

File: examples/docs_snippets/docs_snippets_tests/integrations_tests/test_dbt.py [hunk 2]
```diff
@@ -1,39 +1,36 @@
...
-        {
-            "dbt": DbtCliClientResource(
-                project_dir=DBT_PROJECT_PATH,
-                profiles_dir=DBT_PROFILES_DIR,
-            )
-        },
-    )
+    MANIFEST_PATH = Path(__file__).parent / "manifest.json"
+    with open(MANIFEST_PATH) as f:
+        manifest = json.load(f)
```
Comment (rexledesma) [inline examples/docs_snippets/docs_snippets_tests/integrations_tests/test_dbt.py:26]: nit
```suggestion
    manifest = json.loads(MANIFEST_PATH.read_bytes())
```

**Review by rexledesma — CHANGES_REQUESTED** [review]

File: docs/content/integrations/dbt/reference.mdx [hunk 3]
```diff
@@ -102,28 +102,39 @@ defs = Definitions(
...
 
-Once you have your dbt assets, you can define a job that runs some or all of these assets on a schedule:
+Once you have your dbt assets, you can define a job to materialize a selection of these assets on a schedule.
 
-```python startafter=start_schedule_assets endbefore=end_schedule_assets file=/integrations/dbt/dbt.py dedent=4
-from dagster import ScheduleDefinition, define_asset_job, Definitions
+### Scheduling jobs that contain only dbt assets
 
-run_everything_job = define_asset_job("run_everything", selection="*")
+In this example, we use the <PyObject module="dagster_dbt" object="build_schedule_from_dbt_selection" /> function to create a job, `"daily_dbt_models"`, as well as a schedule which will execute this job once a day. We define the set of models we'd like to execute using [dbt's selection syntax](https://docs.getdbt.com/reference/node-selection/syntax#how-does-selection-work), in this case selecting only the models with the tag "daily".
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt/reference.mdx:109]:
```suggestion
In this example, we use the <PyObject module="dagster_dbt" object="build_schedule_from_dbt_selection" /> function to create a job, `daily_dbt_models`, as well as a schedule which will execute this job once a day. We define the set of models we'd like to execute using [dbt's selection syntax](https://docs.getdbt.com/reference/node-selection/syntax#how-does-selection-work), in this case selecting only the models with the tag `daily`.
```

No quotes needed around these.

File: docs/content/integrations/dbt/reference.mdx [hunk 4]
```diff
@@ -102,28 +102,39 @@ defs = Definitions(
...
-    ],
+daily_dbt_assets_schedule = build_schedule_from_dbt_selection(
+    [my_dbt_assets],
+    job_name="daily_dbt_models",
+    cron_schedule="@daily",
+    dbt_select="tag:daily",
+)
+```
+
+### Scheduling jobs that contain both dbt assets and non-dbt assets
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt/reference.mdx:122]:
```suggestion
### Scheduling jobs that contain dbt assets and non-dbt assets
```

Suggesting we remove `both` from this heading to cut down on length a little bit.

File: docs/content/integrations/dbt/reference.mdx (hunk 1 above)
Comment (OwenKephart) [reply docs/content/integrations/dbt/reference.mdx:137]: Ah yeah fair enough

**Review by rexledesma — APPROVED** [review]

**Pull request merged**